test/e2e/gmcq.cy.js

```diff
@@ -1,26 +1,26 @@
 describe('Graphic Multiple Choice Question', function () {
   beforeEach(function () {
-    cy.getData()
+    cy.getData();
     cy.visit('/');
   });
 
   it('should display the gmcq component', function () {
-    const gmcqComponents = this.data.components.filter((component) => component._component === 'gmcq')
-    gmcqComponents.forEach((gmcqComponent) => {
+    const gmcqComponents = this.data.components.filter(component => component._component === 'gmcq');
+    gmcqComponents.forEach(gmcqComponent => {
       cy.visit(`/#/preview/${gmcqComponent._id}`);
-      const bodyWithoutHtml = cy.helpers.stripHtml(gmcqComponent.body)
-      cy.testContainsOrNotExists('.gmcq__body', bodyWithoutHtml)
-      
-      cy.testContainsOrNotExists('.gmcq__title', gmcqComponent.displayTitle)
-      cy.testContainsOrNotExists('.gmcq__instruction', gmcqComponent.instruction)
-      cy.get('.gmcq-item').should('have.length', gmcqComponent._items.length)
-      gmcqComponent._items.forEach((item) => {
-        cy.testContainsOrNotExists('.gmcq-item__text', item.text)
-      })
+      const stripHtml = cy.helpers.stripHtml;
+      cy.testContainsOrNotExists('.gmcq__body', stripHtml(gmcqComponent.body));
+      cy.testContainsOrNotExists('.gmcq__title', stripHtml(gmcqComponent.displayTitle));
+      cy.testContainsOrNotExists('.gmcq__instruction', stripHtml(gmcqComponent.instruction));
+
+      cy.get('.gmcq-item').should('have.length', gmcqComponent._items.length);
+      gmcqComponent._items.forEach(item => {
+        cy.testContainsOrNotExists('.gmcq-item__text', item.text);
```

Comment:
Please check all of the tested properties for the `compile` helper. That's where the json data is processed as a handlebars template and can produce html.

https://github.com/adaptlearning/adapt-contrib-gmcq/blob/eeadd8f17e1752763a62f3b014d5234380e6b1d0/templates/gmcq.jsx#L127